application/pr/viewer.jsx

```diff
@@ -43,9 +47,28 @@ export default class PRViewer extends React.Component {
                                         })
                                        ));
             })
-            .then(() => this.setState({groupDetails, status: "ready"}))
+            .then(() => {
+                return fetch(pp + "api/team-contact-of", { credentials: "include"})
+                    .then(utils.jsonHandler)
+                    .then((data) => {
+                        if (!data.hasOwnProperty('error')) {
+                            teamcontact = groupDetails.some((g) => {
+                                return data.map((wg) => { return wg.title; }).includes(g.name);
+                            });
+                        }
+                    });
+            })
+            .then(() => {
+                return fetch(pp + "api/repos/" + [owner, shortName].join("/"), { credentials: "include"})
```

Comment:
nit: `=> { return fetch` could be simplified in `fetch`

Reply:
addressed: https://github.com/w3c/ash-nazg/pull/163/commits/3491667e19f54f1d4a3e43e40c05b151d66203a0